README.md

```diff
@@ -4,10 +4,13 @@
 
 - **Version 2**
 
-  - 2.0.7: 
+  - 2.0.8:
+    - added active status to primary topnav
+    - added active status to secondary topnav
```

Comment:
You can use the actual component name and capital letters, like so: _Added active status to PrimaryTopNav_.